model/kubernetes/kubernetes_cluster.rb

```diff
@@ -45,6 +48,14 @@ def sshable
     cp_vms.first.sshable
   end
 
+  def kubectl(cmd)
+    cp_vms.first.sshable.cmd("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf #{cmd}")
+  end
+
+  def kubectl_s(session, cmd)
+    session[:ssh_session].exec!("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf #{cmd}")
```

Comment:
It would be nice if we can avoid the duplication. What about something like this:

```ruby
def kubectl(cmd, session: cp_vms.first.sshable.start_fresh_session)
  session.exec!("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf #{cmd}")
end
```